app.css

```diff
@@ -36,3 +36,37 @@ body {
 .head-wrapper header ul li:hover {
     border-bottom: 1px solid #4cc5e2;
 }
+
+/*Slick*/
+.single-item {
+  direction: ltr;
+  margin: auto;
+  margin-top:80px;
+  height:600px;
+  width: 960px;
```

Comment:
Same.. 
I told u, that it didn't worked out with slick for me, and i need your help, so all the rules here are experimental